BarrysPeanutsTemporalJava/src/main/java/barryspeanuts/model/Address.java

```diff
@@ -0,0 +1,55 @@
+package barryspeanuts.model;
+
+public class Address {
+  private String address1;
+  private String address2;
+  private String city;
+  private String stateProvince;
+  private String zipRegionCode;
+  private String country;
+
+  /*
+  Add a parameterless constructor and setters to avoid complaints
+  by the serializer.
+  */
```

Comment:
I wouldn't say "avoid complaints", but rather "make it serializable". Most Java serialization requires a parameterless constructor. Also, most POJOs have setters though you can rely on the fact that our serializer sets private fields if it has to.

Reply:
Addressed